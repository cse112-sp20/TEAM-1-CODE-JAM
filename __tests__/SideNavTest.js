import "babel-polyfill";
const chrome = require("sinon-chrome");
window.chrome = chrome;

describe("need to take out snapshot", () => {
  test("delete later", () => {});
});
/*eslint-disable*/
// Function to find DOM components recursively
function findDOMComponent(component, searchName) {
  const child = Array.isArray(component) ? component[0] : component;

  console.log(child);

  if (!component.children || !component.children.length) {
    return null;
  }
  if (child.props.className && child.props.id) {
    return (
      child.props.className === searchName || child.props.id === searchName
    );
  } else if (child.props.className) {
    return child.props.className === searchName;
  } else {
    return child.props.id === searchName;
  }

  const findComponent = child.children.find((child) => {
    return !!findDOMComponent(child, searchName);
  });

  return findComponent;
}
