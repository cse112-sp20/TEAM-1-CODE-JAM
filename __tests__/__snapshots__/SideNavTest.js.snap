// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing side navigation bar 1`] = `
<div
  className="app"
>
  <div>
    <nav
      id="nav-background"
    >
      <div
        className="nav-wrapper"
      >
        <a
          className="brand-logo"
          href="#"
        >
          Team 1
        </a>
        <ul
          className="right"
        >
          <li>
            <a
              href="#"
            >
              About
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
  <div
    className="row"
  >
    <div
      className="col s1"
      id="side"
    >
      <div
        className="SideNav"
      >
        <a
          aria-current="page"
          className="btn tooltipped waves-effect waves-light card-panel center-align active"
          data-position="right"
          data-tooltip="Home"
          href="/"
          id="sidenav-background"
          onClick={[Function]}
          style={Object {}}
        >
          <i
            className="white-text material-icons"
          >
            home
          </i>
        </a>
        <a
          aria-current={null}
          className="btn tooltipped waves-effect waves-light card-panel center-align"
          data-position="right"
          data-tooltip="Teams"
          href="/teams"
          id="sidenav-background"
          onClick={[Function]}
        >
          <i
            className="white-text material-icons"
          >
            group_add
          </i>
        </a>
        <a
          aria-current={null}
          className="btn tooltipped waves-effect waves-light card-panel center-align"
          data-position="right"
          data-tooltip="Timeline"
          href="/timeline"
          id="sidenav-background"
          onClick={[Function]}
        >
          <i
            className="white-text material-icons"
          >
            access_time
          </i>
        </a>
        <a
          aria-current={null}
          className="btn tooltipped waves-effect waves-light card-panel center-align"
          data-position="right"
          data-tooltip="Charts"
          href="/charts"
          id="sidenav-background"
          onClick={[Function]}
        >
          <i
            className="white-text material-icons"
          >
            insert_chart
          </i>
        </a>
      </div>
    </div>
    <div
      className="col s11"
      id="component"
    >
      <div
        className="card"
      >
        <div
          className="card-content blue-grey-text"
          id="card-content"
        >
          <span
            className="blue-grey-text card-title"
          >
            
            <span
              className="badge"
            >
              Team Code: 
              
            </span>
          </span>
        </div>
        <div
          className="card-tabs"
        >
          <ul
            className="tabs tabs-fixed-width"
          >
            <li
              className="tab"
            >
              <a
                href="#teamMembers"
              >
                Team Members
              </a>
            </li>
            <li
              className="tab"
            >
              <a
                href="#timeline"
              >
                Timeline
              </a>
            </li>
            <li
              className="tab"
            >
              <a
                href="#charts"
              >
                Charts
              </a>
            </li>
          </ul>
        </div>
        <div
          className="card-content grey lighten-4"
          id="card-content"
        >
          <div
            id="teamMembers"
          >
            <ul
              className="collection"
            />
          </div>
          <div
            id="timeline"
          >
            <div>
              <div
                class="container"
              >
                <div
                  class="timeline"
                  id="timeline"
                />
              </div>
            </div>
          </div>
          <div
            id="charts"
          >
            <div
              chartData={
                Object {
                  "datasets": Array [
                    Object {
                      "backgroundColor": "rgba(11, 113, 126, 0.7)",
                      "data": Array [
                        10,
                        20,
                        5,
                        8,
                        10,
                      ],
                      "label": "My Contributions",
                      "type": "bar",
                    },
                    Object {
                      "backgroundColor": Array [
                        "rgba(255,99,132,0.6)",
                        "rgba(54,162,235,0.6)",
                        "rgba(255,206,86,0.6)",
                        "rgba(75,192,192,0.6)",
                        "rgba(153,102,255,0.6)",
                      ],
                      "data": Array [
                        30,
                        40,
                        50,
                        60,
                        20,
                      ],
                      "label": "Total Contributions",
                      "type": "bar",
                    },
                  ],
                  "labels": Array [
                    "Monday",
                    "Tuesday",
                    "Wednesday",
                    "Thursday",
                    "Friday",
                  ],
                }
              }
              className="chart"
              id="chart"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
